x-pack/plugins/cases/public/components/actions/assignees/edit_assignees_flyout.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import type { AppMockRenderer } from '../../../common/mock';
+import { createAppMockRenderer } from '../../../common/mock';
+import { basicCase } from '../../../containers/mock';
+import { waitForComponentToUpdate } from '../../../common/test_utils';
+import { EditAssigneesFlyout } from './edit_assignees_flyout';
+import { waitFor } from '@testing-library/react';
+
+jest.mock('../../../containers/user_profiles/api');
+
+describe('EditAssigneesFlyout', () => {
```

Comment:
Is it worth adding a test for the expected functionality when hitting save in the flyout when no changes have been made e.g. deselect and the reselect the same user and then hit save. Currently it seems we make a request for this 'noop' update, which is different I think to the behavior in the Case view page.

Reply:
This is tested here: https://github.com/elastic/kibana/pull/146907/files#diff-17a1d7e7f9359bbfd8de09fbfe4235f089bd566b38a69f0a1cd55add392a3df3R230. We do not make a request for this update even though we so the toaster to the user. The reason is to avoid confusion and to avoid handling different toasters for different scenarios.